includes/Dokan.php

```diff
@@ -51,10 +51,12 @@ public function manager_permission( $valid ) {
      * @return void
      */
     public function frontend_permissions( $valid ) {
-        if ( dokan_is_user_seller( dokan_get_current_user_id() ) ) {
-            if ( dokan_is_seller_enabled( dokan_get_current_user_id() ) ) {
+        if ( dokan_is_user_seller( get_current_user_id() ) ) {
```

Comment:
can we make these two conditions in one line?

Reply:
Yes we can.

Reply:
Please update.

Reply:
updated